builder/dockerfile/dispatchers.go

```diff
@@ -348,11 +348,15 @@ func dispatchRun(d dispatchRequest, c *instructions.RunCommand) error {
 	if err := d.builder.containerManager.Run(d.builder.clientCtx, cID, d.builder.Stdout, d.builder.Stderr); err != nil {
 		if err, ok := err.(*statusCodeError); ok {
 			// TODO: change error type, because jsonmessage.JSONError assumes HTTP
+			msg := fmt.Sprintf(
```

Comment:
It seems preferable to wrap this in `if err.Error() == ""` or equivalent, but I realize that requires declaring `msg` explicitly outside of the conditional statements. I guess it's fine as is, since the common path is surely no error.